### ISSUE TITLE
Function names aren't colored

### ISSUE THREAD

**Issue by Eternex49** [srcery-colors/srcery-vim#82]
As the title states, function names in C and C++ are left white whilst they should be yellow.

I've checked the installed vimscript file and, indeed, function names are set to be yellow.

Using: Neovim 0.7.2

**Comment by roosta** [comment]
Hi, doesn't seem vim defines any C function highlight group, I get back nothing when trying `synIDattr`, are you using some other syntax file that didn't come with vim?

**Comment by Eternex49** [comment]
Yet in the example image the function names are infact colored in yellow.
No, I'm not using some other synatx file that didn't come with vim, anyway.

**Comment by roosta** [comment]
That they are...huh. I code a fair bit in c, and I don't remember them being yellow, but I took the screenshots. I checked the same source file, and sure enough, its white now. Honestly not sure why that is.
I checked the syntax file included with vim and it doesn't seem to define any groups for functions, you'd have to bake your own, which would be tricky to target in srcery.

https://github.com/vim/vim/blob/master/runtime/syntax/c.vim
